warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -1314,11 +1314,6 @@ public void progress() {
     }
   }
 
-  @Override
-  public boolean isAuthenticated() {
-    return null != this.getAttribute(ATTRIBUTE_TOKEN);
-  }
-
```

Comment:
If an existing extension uses isAuthenticated(), it may be better to raise an explicit exception, or to return false. (with a comment to explain why in the code)
It will guide the developer to update its extension the right way.

The best way to avoid to break anything legacy, is to check ".cap:limits", which is nearly equivalent behaviour compared to v2.x AUTHENTICATE.